cli/util/color.go

```diff
@@ -0,0 +1,9 @@
+package cliutil
+
+import (
+	"os"
+
+	"github.com/mattn/go-isatty"
+)
+
+var DefaultColorUse = isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd())
```

Comment:
Can we also have an env var for this? (`LOTUS_COLOR=1`, or sth like that, unless there is already a standard env var for that kind of thing)

Just `isatty.IsTerminal` will return false when running stuff in `watch --color ..` for example.

Reply:
So... we could introduce `LOTUS_COLOR=1` *OR* we could recognize `GOLOG_LOG_FMT=color` which is [a thing from `ipfs/go-log`](https://github.com/ipfs/go-log/pull/116#issuecomment-873779808)

I can go either way, just suggesting an alternative for smaller amount of options to keep track of...

@magik6k say the word

Reply:
@Stebalien please be weighing in as well, I surely missed other stuff here too ;)

Reply:
`GOLOG_LOG_FMT` works for me

Reply:
@magik6k please be checking https://github.com/filecoin-project/lotus/pull/6696/commits/4dd59b566ced